modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/components/button/Button.es.js

```diff
@@ -36,6 +36,7 @@ const Button = (props) => {
 			data-title={tooltip}
 			ref={forwardRef}
 			symbol={symbol}
+			monospaced={symbol ? true : false}
```

Comment:
Hey @interaminense, just wanted to let you know about this change I made to some code you had written.

`ClayButtonWithIcon` now adds monospaced as a prop and defaults to true. Because you were passing in monospaced, I updated it to pass true for any ClayButtonWithIcon and false for just ClayButton

Let me know if you have any questions about this

Reply:
Hey @bryceosterhaus, thank you for let me know about it. That's ok for me :) 
Just to you know, the app builder team will refactor all the components that are using Buttons to use the ClayButton and then remove it.

Reply:
`? true : false` always strikes me as a bit of a smell because it reads tautologically as "if true then true otherwise false".

`!!symbol` would probably be the idiomatic way to coerce to boolean.